test/fixtures/config/forms.js

```diff
@@ -0,0 +1,9 @@
+import faker from '@roundingwellos/faker';
+
+export default () => {
+  return {
+    id: faker.datatype.uuid(),
+    name: `${ faker.company.bsBuzz() } ${ faker.company.catchPhraseNoun() }`,
```

Comment:
I think we need to add the `created_at` `updated_at` `submitted_at` timestamps here

Reply:
Added 👍